### PR TITLE
Function arguments by forwarding reference - issue 59

### PR DESCRIPTION
**Pull request by AndresSanchezCuadrado**
Changed 'const &' with '&&' except in composed pipeline

### CODE REVIEW

**Review by jdgarciauc3m — CHANGES_REQUESTED** [review]
Use forwarding references in pipelines too.

File: include/ppi_omp/pipeline_omp.h [hunk 1]
```diff
@@ -266,6 +264,7 @@ void pipeline(parallel_execution_omp p, FuncIn const & in, Stages ... sts ) {
                 }
             }
             //Create next stage
+            //stages(p, q, std::forward<Stages>(sts) ... );
```
Comment (AndresSanchezCuadrado) [inline include/ppi_omp/pipeline_omp.h:267]: If I use the forward at this point, there are compilation problems for the examples pipeline+filter_test_ppi_example_OMP

**Review by jdgarciauc3m — APPROVED** [review]
This solves issue #59

**Review by jdgarciauc3m — CHANGES_REQUESTED** [review]
Please, remove thrust files from this pull request.

**Review by jdgarciauc3m — APPROVED** [review]
Finally solved issue #59

**Pull request merged**